roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install RPM key for StackStorm
+  become: yes
+  rpm_key:
+    state: present
+    key: https://packagecloud.io/StackStorm/stable/gpgkey
+
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: stackstorm
+    description: Stackstorm Repo
+    file: st2
```

Comment:
Here is yum repo file I got for `CentOS7` when adding st2 packagecloud repo: 

```sh
[vagrant@centos7 ~]$ cat /etc/yum.repos.d/StackStorm_stable.repo
[StackStorm_stable]
name=StackStorm_stable
baseurl=https://packagecloud.io/StackStorm/stable/el/7/$basearch
repo_gpgcheck=1
gpgcheck=0
enabled=1
gpgkey=https://packagecloud.io/StackStorm/stable/gpgkey
sslverify=1
sslcacert=/etc/pki/tls/certs/ca-bundle.crt
metadata_expire=300
```

A lot of security checks here by PackageCloud. I think we should include more `yum_repository` options where it's possible and looks easy.

Additionally, it would be nice to follow the same naming for the file and repo.